js/addressfinder/src/magento.js

```diff
@@ -248,13 +248,16 @@ export default class Magento {
 
         this.debug('Setting value for mapping', key, value);
 
-        this.getMapping(key).value = value;
+        let mapping = this.getMapping(key)
```

Comment:
Theres a few lines in this file missing semi colons, may be worth adding these to keep the file consistent.